.eslintrc

```diff
@@ -1,13 +1,12 @@
 {
   "extends": [
-    "prettier",
```

Comment:
This is not needed, it's actually added by `""plugin:prettier/recommended",` which in turn should be the last item in the `"extends" array